beacon-chain/rpc/eth/beacon/handlers.go

```diff
@@ -1893,6 +1895,111 @@ func (s *Server) GetProposerLookahead(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// GetBlobs retrieves blobs for a given block id.
+func (s *Server) GetBlobs(w http.ResponseWriter, r *http.Request) {
```

Comment:
Why is this not in `beacon-chain/rpc/eth/blob/handlers.go`?

Reply:
moved